### ISSUE TITLE
Is Scene box working correctly?

### ISSUE THREAD

**Issue by snarb** [nerfstudio-project/nerfstudio#967]
I have noticed that changing 

`scene_box = SceneBox(aabb=torch.tensor([[-1.5, -1.5, -1.5], [1.5, 1.5, 1.5]], dtype=torch.float32))`

in the blender_dataparser.py to 

`scene_box = SceneBox(aabb=torch.tensor([[-.005, -.005, -.005], [.005, .005, .005]], dtype=torch.float32))`

does not affects the training on the Lego scene significantly. This looks suspicious to me. Maybe this just means that normalization from 0 to 1 doesn't meters a lot.

**Comment by tancik** [comment]
Since there is a `spatial_distortion` in the nerfacto model, the scene box is never used. https://github.com/nerfstudio-project/nerfstudio/blob/009e21be248418028e6303ecde593a20ef88fa87/nerfstudio/fields/nerfacto_field.py#L199-L204
It is worth noting that the nerfacto model isn't really setup to work with the blender scenes.

**Comment by snarb** [comment]
Clear. Thanks . Does it make sense to tune nerfakto for synthetic scenes or instant ngp will work thr same or better after all?
